theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-sub-widgets/trace-explorer-placeholder-widget.tsx

```diff
@@ -54,6 +55,12 @@ export class TraceExplorerPlaceholderWidget extends ReactWidget {
         this.id = TraceExplorerPlaceholderWidget.ID;
         this.title.label = TraceExplorerPlaceholderWidget.LABEL;
         this.update();
+        signalManager().on(Signals.OPEN_NEW_TRACE, this.handleOpenTrace);
```

Comment:
@PatrickTasse @bhufmann any suggestions? I catch the signal in the placeholder widget because the button would be in that component and the widget already has a method to handle opening of traces. An alternative could be to manage the signal in a parent component like trace-viewer.tsx or trace-explorer-widget.tsx and move handleOpenTrace functionality to a utils file or service